conans/test/functional/toolchains/gnu/test_pkgconfigdeps.py

```diff
@@ -621,5 +622,38 @@ def package_info(self):
     client.save({"conanfile.txt": "[requires]\nhello/1.0\n"}, clean_first=True)
     client.run("install . --build=missing -g PkgConfigDeps")
     client.run_command("PKG_CONFIG_PATH=$(pwd) pkg-config --cflags hello")
-    assert r'flag2=\"my flag2\" flag1=\"my flag1\" -DUSER_CONFIG=\"user_config.h\" -DOTHER=\"other.h\"' in client.out
+    assert r'flag2=\"my flag2\" flag1=\"my flag1\" ' \
+           r'-DUSER_CONFIG=\"user_config.h\" -DOTHER=\"other.h\"' in client.out
 
+
+def test_pkgconfigdeps_with_build_and_test_requires():
```

Comment:
I would have preferred this is a faster integration test that checks the .pc is produced. Functional tests are slow and heavy, we want to keep our test suite as light as possible.

Reply:
I tend to say that a lot of tests from this module are integration ones...

Reply:
It depends if they actually need the ``pkg_config`` tools or not. If not, yes, they can be moved to ``integration``. But not now, better in 2.0 only, and after we have stabilized and no need to do more develop->develop2 merges.